frontend/src/journal/layouts/Sidebar.tsx

```diff
@@ -0,0 +1,71 @@
+import { NavLink } from "react-router-dom";
+import { SignOutButton } from "@clerk/clerk-react";
+import { useGetJournalsQuery } from "@/graphql/generated";
+import { MenuLink } from "../types/sidebar";
+
+import sidebar from "../../assets/sidebar.svg";
+import signout from "../../assets/signout.svg";
+import add from "../../assets/add.svg";
+import journal from "../../assets/journal.svg";
+
+const menuLinks: MenuLink[] = [
+  { path: "/create-journal", name: "New Personal Journal", img: add },
+];
+
+export default function Sidebar(): JSX.Element {
+  const { data } = useGetJournalsQuery();
+
+  const totalJournals = data?.getJournals?.length || null;
+
+  return (
+    <div className="w-[220px] flex flex-col justify-between h-[100vh] bg-[#f9f9f9]">
+      <div>
+        <div>
+          <div className="p-2 border-b">
+            <img src={sidebar} alt="" />
```

Comment:
It's a good practice to always add alt description to your images

This is helpful for accessibility features, like screen readers, helpers for partially blind people, etc

Reply:
Replace it by Icon library